src/lib/autoinstall/autosetup_helpers.rb

```diff
@@ -198,6 +198,15 @@ def autosetup_country
       Yast::Profile.remove_sections("language") if Yast::Profile.current.key?("language")
     end
 
+    # Invokes autoyast setup for firewall
+    def autosetup_firewall
+      return if !Yast::Profile.current["firewall"]
+
+      log.info("Importing Firewall settings from AY profile")
+      Yast::WFM.CallFunction("firewall_auto", ["Import", Yast::Profile.current["firewall"]])
+      Yast::WFM.CallFunction("firewall_auto", ["Write"])
```

Comment:
The idea was to remove the section once writing as we are doing with others section, but it is still missing.

Reply:
sorry this PR is a little obsolete, my fault